db/migrate/20200521150645_umm_s_1_3_data_migration.rb

```diff
@@ -0,0 +1,217 @@
+class UmmS13DataMigration < ActiveRecord::Migration[5.2]
```

Comment:
I'm not sure how much this matters, but I feel that the methods here could use a lot more comments. This is a confusing change and having more comments could give more context. That being said, migrations are usu a one time thing and isn't code that some future developer will need to modify, and if there's an issue I assume you're going to fix it.